### PR TITLE
CNV-30339: Updating storage requirements for CNV installation

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.14+

Issue: [CNV-30339](https://issues.redhat.com//browse/CNV-30339)

Link to docs preview: https://65040--docspreview.netlify.app/openshift-enterprise/latest/virt/install/preparing-cluster-for-virt#storage-requirements_preparing-cluster-for-virt

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: virt/install/preparing-cluster-for-virt.adoc [hunk 1]
```diff
@@ -94,6 +94,13 @@ See xref:../../architecture/architecture-rhcos.adoc#rhcos-about_architecture-rhc
 
 * {rh-storage-first}: You must create a dedicated storage class for Windows virtual machine disks. See link:https://access.redhat.com/articles/6978371[Optimizing ODF PersistentVolumes for Windows VMs] for details.
 
+[NOTE]
+====
+The cluster must have a default storage class set. See xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#persistent-storage-csi-sc-manage[Managing the default storage class]. The default storage class provisioner must support the `ReadWriteMany` (RWX) xref:../../storage/understanding-persistent-storage.adoc#pv-access-modes_understanding-persistent-storage[access mode].
```
Comment (aglitke) [inline virt/install/preparing-cluster-for-virt.adoc:99]: The default storage class _should_ support ReadWriteMany for the best experience.  In 4.15 we will be adding a default storage class for virtualization (see https://issues.redhat.com/browse/CNV-32030).  We may want to say something like (if you are unsure which storage class to use check with your storage vendor).

Comment (sjhala-ccs) [reply virt/install/preparing-cluster-for-virt.adoc:99]: @aglitke Does rewriting the above paragraph in the following manner make sense:

``
You must specify a default storage class for the cluster. See xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#persistent-storage-csi-sc-manage[Managing the default storage class]. For optimal performance, if the default storage class provisioner supports the `ReadWriteMany` (RWX) xref:../../storage/understanding-persistent-storage.adoc#pv-access-modes_understanding-persistent-storage[access mode], use the RWX mode for the associated persistent volumes.
``

Comment (aglitke) [reply virt/install/preparing-cluster-for-virt.adoc:99]: looks good.  Thanks!

**Review by kgoldbla — COMMENTED** [review]
lgtm

**Pull request merged**